_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,235 @@
+
+Values from the command line argument have higher priority than the environment file. For example, if you add filename as *test* in *.env* file and also add `--filename report` command option, the generated report's name will be *report*.
+{: .note }
+
+### Example: Request a PNG report with environment variables set
+
+The following command requests a dashboard report with basic authentication in png format.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --format png --auth basic --credentials admin:admin
+```
+
+Upon success, the report will download to the current directory.
+
+### Use Amazon SES to request an email with a report attachment
+
+To use Amazon SES as the email transport mechanism, the following prerequisites apply:
+
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily. The sender's email address must be verified on [Amazon SES](https://aws.amazon.com/ses/).
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.
+ [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) is required to interact with Amazon SES.
+ [Configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) basic settings used by AWS CLI.
+- SES transport requires the `ses:SendRawEmail` role.
+
+```
+{
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "ses:SendRawEmail",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+The following command requests an email with the report attached:
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport ses --from <sender_email_id> --to <recipient_email_id>
+```
+
+The following command uses default values for all other options. You can also set `OPENSEARCH_FROM`, `OPENSEARCH_TO`, and `OPENSEARCH_TRANSPORT` in .env file and use following command.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d
+```
+
+To modify the body of your email, you can simply edit *index.hbs* file.
+
+### Example: Send a report to email with SMTP
+
+To send a report through email with SMTP transport, you need to set the options `OPENSEARCH_SMTP_HOST`, `OPENSEARCH_SMTP_PORT`, `OPENSEARCH_SMTP_USER`, `OPENSEARCH_SMTP_PASSWORD`, `OPENSEARCH_SMTP_SECURE` in your .env file.
+
+Once the transport options are set in .env file, you can send the email using the following command.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport smtp --from <sender_email_id> --to <recipient_email_id>
+```
+
+You can choose to set options using *.env* file or the command line argument values in any combination. Make sure to specify all required values to avoid getting errors.
```

Comment:
```suggestion
You can choose to set options using either your *.env* file or the command line argument values in any combination. Make sure to specify all required values to avoid errors.
```